core/makeswift/components/footer/register.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link, List, Shape, TextInput } from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { MakeswiftFooter } from './client';
+
+export const COMPONENT_TYPE = 'catalyst-makeswift-footer';
+
+runtime.registerComponent(MakeswiftFooter, {
+  type: COMPONENT_TYPE,
+  label: 'MakeswiftFooter (private)',
```

Comment:
Do we want to put `(private)` in the label? This will show up in the Makeswift UI in certain places like breadcrumbs, etc.

Reply:
The embedded component label overrides this one:

<img width="791" alt="Screenshot 2024-11-06 at 4 37 12 PM" src="https://github.com/user-attachments/assets/2808f1b4-57de-4612-9e7e-1a233164d723">

I'm not married to the `(private)` suffix, but it does help differentiate these "hidden" components more easily in Redux DevTools/logs.